versioned_docs/version-8.7/reference/release-notes/870.md

```diff
@@ -200,7 +200,7 @@ This [alpha release](/reference/release-policy.md) contains a known issue where
 
 You can now use a single Query API in the Camunda 8 REST API to find process and decision data instead of using multiple component APIs.
```

Comment:
💭 This page is going to be confusing for people, I could imagine. The alphas until 8.7.0-alpha3 contributed to what now is 8.8, actually. The stuff mentioned there is not reflected (entirely) in 8.7.0 anymore. That is a tricky page now. One thought here: maybe we move the alphas until alpha3 to 8.8 and explain there that those changes contributed to 8.8 and not 8.7. Here in 8.7 we start listing from alpha4. In the end, this is what we ACTUALLY did so maybe this should also be reflected in the docs as well.

Reply:
//cc @akeller

Reply:
Let's handle this outside the scope of this PR. @conceptualshark has been shifting things around to reflect what is and isn't delivered in 8.7 vs. 8.8, including things delivered and shifted in alphas.